spec/factories/user_session.rb

```diff
@@ -0,0 +1,37 @@
+Dir[File.join(__dir__, '../../app/models/wizard/steps/', '*.rb')].sort.each { |file| require_relative file }
+
+FactoryBot.define do
+  factory :user_session do
```

Comment:
How do you feel about adding traits like e.g gb_to_ni and then you prepopulate the answers as well rather than leaving them all nil by default? Just a thought...

Reply:
It's exactly what I planned to do as a follow on PR - using human-readable traits to define the session